openmc/universe.py

```diff
@@ -50,7 +50,7 @@ class Cell(object):
         Unique identifier for the cell
     name : str
         Name of the cell
-    fill : Material or Universe or Lattice or 'void'
+    fill : Materials or Universe or Lattice or 'void'
```

Comment:
Really should be "Material or Universe or Lattice or 'void' or Iterable of Material".